src/pages/proposal.tsx

```diff
@@ -430,7 +404,7 @@ export default function ProposalPage() {
           )}
           {showSurveyResults && (
             <SurveyResults
-              votes={votes}
+              votes={highQualityVotes ?? null}
```

Comment:
Why are we including only the high vp votes for the survey results? 
Are bots also sending a survey to snapshot?

Reply:
I guess it makes sense if we only count the high quality votes for displaying the proposal's partial results...